lms/static/js/learner_dashboard/spec/program_details_view_spec.js

```diff
@@ -497,6 +508,25 @@ describe('Program Details Header View', () => {
     };
     const data = options.programData;
 
+    const testSubscription = (state, heading, body) => {
```

Comment:
should it be renamed to `testSubscriptionStatus`?

Reply:
sure